src/SB/Game/zMain.cpp

```diff
@@ -635,8 +755,62 @@ void zMainMemCardSpaceQuery()
     S32 status;
 }
 
+void zMainMemCardQueryPost(S32 needed, S32 available, S32 neededFiles, S32 unk0)
+{
+    RwCamera* cam = 0;
+    RwRGBA* colour = 0;
+    RwInt32 clearMode = 3;
+
+    cam = iCameraCreate(0x280, 0x1e0, 0);
+    RwCameraClear(cam, colour, clearMode);
+    RwCameraBeginUpdate(cam);
+    render_mem_card_no_space(needed, available, neededFiles, unk0);
+    RwCameraEndUpdate(cam);
+    RwCameraShowRaster(cam, 0, 1);
+    iCameraDestroy(cam);
+}
+
+void zMainMemCardRenderText(const char* a, bool enabled)
+{
+    RwCamera* cam = 0;
+    RwRGBA* colour = 0;
+    RwInt32 clearMode = 3;
+
+    cam = iCameraCreate(0x280, 0x1e0, 0);
+    RwCameraClear(cam, colour, clearMode);
+    RwCameraBeginUpdate(cam);
```

Comment:
These are meant to specify the screen dimensions 640 x 480 - please replace the hex values with their integer equivalents.